lib/generators/doorkeeper/previous_refresh_token_generator.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails/generators/active_record'
+
+module Doorkeeper
+  class PreviousRefreshTokenGenerator < Rails::Generators::Base
```

Comment:
Something must have changed since the time it was originally created.
Will move module directly into class definition.